lib/src/screens/settings/display_settings_page.dart

```diff
@@ -66,13 +69,14 @@ class DisplaySettingsPage extends BasePage {
               selectedItem: _displaySettingsViewModel.languageCode,
               onItemSelected: _displaySettingsViewModel.onLanguageSelected,
               images: LanguageService.list.keys
-                  .map((e) => Image.asset("assets/images/flags/${LanguageService.localeCountryCode[e]}.png"))
+                  .map((e) => Image.asset(
+                      "assets/images/flags/${LanguageService.localeCountryCode[e]}.png"))
                   .toList(),
               matchingCriteria: (String code, String searchText) {
                 return LanguageService.list[code]?.toLowerCase().contains(searchText) ?? false;
               },
             ),
-            if (DeviceInfo.instance.isMobile)
+            if (ResponsiveLayoutUtil.instance.isMobile)
```

Comment:
This should check by both, Plartform and UI, so that users can't shrink their Mac window to Mobile view, change the theme, then go back to Mac view
i.e
```dart
if (ResponsiveLayoutUtil.instance.isMobile && DeviceInfo.instance.isMobile)
```